util/buildRelease/gen_release

```diff
@@ -94,7 +94,7 @@ if (exists($ENV{"CHPL_GEN_RELEASE_NO_CLONE"})) {
     }
 
     print "[gen_release] Creating build workspace with git archive...\n";
-    SystemOrDie("cd $rootdir && git archive --format=tar HEAD | (cd $archive_dir && tar -xf -)");
+    SystemOrDie("cp --archive $rootdir $archive_dir");
```

Comment:
Something about this change seems to be breaking my ability to run `gen_release` on my Mac.  I get:

```
+ cp --archive /var/folders/69/jbk0y4q95lq8xhhwpqgwf_rh000bxs/T/chapel-release.bradc.deleteme.D7yQE/chpl_home /var/folders/69/jbk0y4q95lq8xhhwpqgwf_rh000bxs/T/chapel-release.bradc.deleteme.D7yQE/chapel-1.20.0 
cp: illegal option -- -
```